tests/visual/landingPage.spec.ts

```diff
@@ -19,3 +19,14 @@ test("landing page", async ({ page }) => {
   await page.goto("/", { waitUntil: "networkidle" });
   await expect(page).toHaveScreenshot();
 });
+
+test("load error", async ({ page }) => {
```

Comment:
I think we should add the FakeTimers to this test, else the activity is going to be shifting every week or so.

Reply:
Good catch. Fixed.